.github/ISSUE_TEMPLATE/Standard.md

```diff
@@ -34,10 +34,11 @@ Where is this issue occurring?
 **Version Number:**
 **Reproducible in staging?:**
 **Reproducible in production?:**
+**Email of affected user:**
 **Logs:** https://stackoverflow.com/c/expensify/questions/4856
 **Notes/Photos/Videos:** Any additional supporting documentation
 **Expensify/Expensify Issue URL:**
 **Issue reported by:**
-**Slack conversation:** 
```

Comment:
NAB; Personally I kinda like these extra spaces after these so you don't have to add them when filling out the template 🤷

Reply:
I think I just have auto-trim whitespace set so it's not possible to save a file without getting rid of the whitespace lol I didn't think about how they might be useful.